### PR TITLE
[RORO-4492][JT] add service dashboard page

### PR DESCRIPTION
**Pull request by tianjianfeng**

### CODE REVIEW

File: app/uk/gov/hmrc/customsservicestatusfrontend/views/HelloWorldPage.scala.html [hunk 1]
```diff
@@ -14,15 +14,56 @@
...
 
 @layout(pageTitle = Some("customs-service-status-frontend")) {
-    <h1 class="govuk-heading-xl">customs-service-status-frontend</h1>
-    <p class="govuk-body">@{messages("service.text")}</p>
-}
+  <h1 class="govuk-heading-xl">@messages("dashboard.h1")</h1>
+  <h2 class="govuk-heading-x">@messages("dashboard.h2")</h2>
+
+ @govukInsetText(InsetText(
+  content = HtmlContent(messages("dashboard.inset", s"<b>${Formatters.instantFormatDate(Instant.now).toString} ${Formatters.instantFormatHours(Instant.now)}</b>", s"""<a href="${uk.gov.hmrc.customsservicestatusfrontend.controllers.routes.HelloWorldController.helloWorld}">""", "</a>"))
```
Comment (tipperba) [inline app/uk/gov/hmrc/customsservicestatusfrontend/views/HelloWorldPage.scala.html:37]: Can you remove all references to hello world? (ie. change the HelloWorldController, the HelloWorldPage view, the helloWorld route etc so they have more appropriate names)

File: conf/messages [hunk 2]
```diff
@@ -1,2 +1,8 @@
-service.name = customs-service-status-frontend
-service.text = This is your new service
+service.name = Customs Service Status Frontend
```
Comment (Mahesh49) [inline conf/messages:1]: this should be `Customs service availability` if this name is for the header

Comment (tianjianfeng) [reply conf/messages]: done

File: app/uk/gov/hmrc/customsservicestatusfrontend/views/HelloWorldPage.scala.html (hunk 1 above)
Comment (tianjianfeng) [reply app/uk/gov/hmrc/customsservicestatusfrontend/views/HelloWorldPage.scala.html]: done

**Review by tipperba — APPROVED** [review]

File: app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+@this(layout: Layout, govukInsetText : GovukInsetText, govukTable : GovukTable)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = Some("customs-service-status-frontend")) {
+  <h1 class="govuk-heading-xl">@messages("dashboard.h1")</h1>
+  <h2 class="govuk-heading-m">@messages("dashboard.h2")</h2>
+
+ @govukInsetText(InsetText(
+  content = HtmlContent(messages("dashboard.inset", s"<b>${Formatters.instantFormatDate(Instant.now).toString} ${Formatters.instantFormatHours(Instant.now)}</b>", s"""<a href="${uk.gov.hmrc.customsservicestatusfrontend.controllers.routes.DashboardController.show}">""", "</a>"))
```
Comment (tipperba) [inline app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html:35]: Can we take out the instantFormatDate part and just show hours, adding in am and pm (in line with Shalini comment?

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Review by elabeca — APPROVED** [review]

**Pull request closed without merging**